src/signals/incident/services/object-flatten/index.js

```diff
@@ -0,0 +1,10 @@
+// based on source: https://30secondsofcode.org/object#flattenobject
+const flattenObject = (obj, prefix = '', separator = '.') =>
+  Object.keys(obj).reduce((acc, k) => {
+    const pre = prefix.length ? prefix + separator : '';
+    if (typeof obj[k] === 'object') Object.assign(acc, flattenObject(obj[k], pre + k, separator));
+    else acc[pre + k] = obj[k];
```

Comment:
Use template literals

Reply:
Disagree, don't think it will improve readability here

Reply:
In my opinion it will. Using `a + b` implies an arithmetic operation instead of a string concatenation.

Reply:
I think I got a nicer solution with a template literal. Should be to your liking. Resolving